Data/Sys/ApprovedInis.json

```diff
@@ -0,0 +1,323 @@
+{
+  "D43J01": {
+		"title" : "ZELDA OCARINA MULTI PACK",
+    "CAB9CED2D904F12CCB21F5B1DE9B5433620C3E13" : "loophack"
+  },
+  "G2BE5G" : {
+		"title" : "Black & Bruised",
+    "7FFF6BDD93713BEDFD23739C32B86153FA19AEA0" : "Disable interlaced rendering"
+  },
+  "G2BP7D" : {
+		"title" : "Black & Bruised",
+    "56E85D7285F10348E1E5354E379918D07E79EDA9" : "Disable interlaced rendering"
+  },
```

Comment:
Something is very messy with the formatting in this file. Please run it through a prettifier or something like that.